src/styles/components/_signInForm.scss

```diff
@@ -0,0 +1,128 @@
+.smilies-img{
+  margin-top: 70px;
+  width: 133px;
+  height: 78px;
+}
+
+.sign-in-title {
+	width: 200;
+	height: 27px;
+  margin-top: 16px;
+	font-family: sans-serif;
+	font-size: 20px;
+	font-weight: bold;
+  display: inline-block;
+	letter-spacing: 3px;
+	color: #000000;
+}
+
+.sign-in-subtitle {
+	width: 292px;
+	height: 32px;
+  display: inline-block;
+  margin-top: 13px;
+  margin-bottom: 0px;
+	font-family: sans-serif;
+	font-size: 18px;
+	font-weight: 600;
+	line-height: 1.8;
+	color: #212121;
+}
+
+.sign-in-info-text {
+	width: 321px;
+	height: 78px;
+	font-family: sans-serif;
```

Comment:
The font-family generally is defined in the html body, it's weird to define for each class